pkg/cli/admin/upgrade/status/mco/layered_node_state.go

```diff
@@ -208,6 +211,10 @@ func isNodeDoneAt(l *LayeredNodeState, pool *mcfgv1.MachineConfigPool) bool {
 
 const (
 	ReasonOfUnavailabilityMCDWorkInProgress = "MCDWorkInProgress"
+
+	ReasonOfUnavailabilityNodeNotReady           = "Node is not ready"
+	ReasonOfUnavailabilityNodeDiskPressure       = "Node is running out disk"
```

Comment:
Let's not deviate from the condition name - not using the condition name prevents the users from recognizing the concepts they may know (or google). "Node has disk pressure".

Reply:
Done